lib/card_generator.rb

```diff
@@ -0,0 +1,21 @@
+require 'pry'
+require './lib/card'
```

Comment:
This `require` IS required because we are creating **new** `Card` objects inside of this class. The `pry` `require` can go though.